doc/developer/guide.md

```diff
@@ -248,6 +248,23 @@ recommendations.
 
 See [Developer guide: submitting and reviewing changes](guide-changes.md).
 
+## Publishing code changes
+
+### crates.io
+
+Before publishing internal Rust crates to `crates.io`(https://crates.io/), be sure to
+indicate that `MaterializeInc` is the reponsible maintainer by running the following
+command:
+```shell
+cargo owner --add github:MaterializeInc:crate-owners
+```
```

Comment:
should we also remove ourselves as owners, so that everything is owned only by the crate-owners group?

Reply:
oh is that possible? If so make sense to add that command here, and then we can hunt everyone down and ask them to run it.

Reply:
Yup, [according to the docs](https://doc.rust-lang.org/cargo/commands/cargo-owner.html) it's 
`cargo owner --remove <login>`

Reply:
Will add today!